main.css

```diff
@@ -136,7 +138,7 @@ body {
     transition: color 0.5s ease;
     color: #FC7700 !important;
 }
-
+/*TODO powtarzające się zbędne stylowania w css'ach na przyklad w pseudoklasach tak jak w nav-linku, moze po zmianie kolejnosci css'a z bootstrapem w headzie w indexie bedzie normalnie dzialalo i ograniczy sie te importanty. poza tym zawsze mozna zwiekszyc sile selektora jezeli bootsrap bedzie mimo wszystko wygrywal ;)*/
```

Comment:
Warto też spojrzeć czy klasa bootstrapa nie daje tylko np. koloru. Jeżeli tak to przy chęci jego zmiany lepiej po prostu stworzyć własną klasę? :)

Reply:
Siły selektora nie będziesz musiał zwiększać jeżeli będzie równa tej z bootstrapa - wystarczy, że selektor napisany przez Ciebie znajdzie się niżej w kodzie - będzie ważniejszy